crates/client/src/client/schema/chain.rs

```diff
@@ -324,10 +324,9 @@ impl TryFrom<GasCosts> for fuel_core_types::fuel_tx::GasCosts {
     fn try_from(value: GasCosts) -> Result<Self, Self::Error> {
         match value.version {
             GasCostsVersion::V1 => {
```

Comment:
This doesn't seem to match, it still says V1 but it's using V2 internally.

Reply:
GraphQL version doesn't need to match the Rust version of the type. It was done to maintain backward compatibility. Because SDKs will throw an error if the version is unknown.

Reply:
Will it get confusing if the versions start diverging a lot over time?

Reply:
When it will diverging a lot, we just can jump straight to the highest=) I was done for backward compatibility until SDKs will handle it gracefully.